doc/technotes/errorHandling.rst

```diff
@@ -0,0 +1,190 @@
+.. _readme-errorHandling:
+
+==============
+Error Handling
+==============
+
+This README describes the error handling feature as currently implemented.
+
+
+Errors
+------
+A base class ``Error`` is defined in the Chapel standard modules.
+
+::
+
+  class Error {
+    var msg: string;
+
+    proc Error(_msg: string) {
+      msg = _msg;
```

Comment:
I'm not so sure if it's worth including some of the details here:
 * that there is a member variable called `msg`
 * that the constructor sets msg = _msg

I think it might be best to just say something like:
```
A base class ``Error`` is defined in the Chapel standard modules. It currently includes an initializer accepting a string argument. That argument is the error message to display if the error is not caught.
```